Packs/RecordedFuture/ReleaseNotes/1_3_0.md

```diff
@@ -0,0 +1,80 @@
+#### Pack
+
+##### Recorded Future Integration
+
+- Note: pack was renamed from "RecordedFuture v2" to "Recorded Future Integration".
+
+#### Integrations
+
+##### Recorded Future v2
+
+- Added the ***recordedfuture-malware-search*** command.
+- Added the ability to get intelligence for malware using the **entity_type** argument and added the ability to fetch
+  analyst notes using the new argument: **fetch_analyst_notes** for ***recordedfuture-intelligence*** command.
+- Made the argument **fetch_related_entities** be optional for ***recordedfuture-intelligence*** command. Default value
+  is "no".
+- Added the **fetch_statuses** integration parameter, which specify alert statuses to include in the fetch. If empty,
+  the default value of 'no-action' will be used.
+- Added the **update_status** integration parameter, which specify whether to update alert status to "pending" after it
+  have been fetched (only for alerts with "no-action" status).
```

Comment:
Please use the display name of the new integration parameters here.

Reply:
Done.